packages/client/hmi-client/src/components/operator/tera-operator-menu.vue

```diff
@@ -9,35 +9,45 @@
 		aria-controls="overlay_menu"
 		severity="secondary"
 	/>
-	<Menu ref="menu" id="overlay_menu" :model="menuItems" :popup="true" />
+	<Menu
+		ref="menu"
+		id="overlay_menu"
+		:model="menuItems"
+		:popup="true"
+		@mouseenter="emit('menu-mouseenter')"
+		@mouseleave="emit('menu-mouseleave')"
+		@focus="() => {}"
+		@blur="() => {}"
```

Comment:
Why are focus/blur specified as no-ops?

Reply:
the type check was asking for them when using `mouseenter` & `mouseleave`.

Reply:
Because that's what they should be used for on an input, the `mouseenter` and `mouseleave` is a last resort.